x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/host_isolation_exceptions_list.tsx

```diff
@@ -50,6 +50,7 @@ type HostIsolationExceptionPaginatedContent = PaginatedContentProps<
   typeof ExceptionItem
 >;
 
+/* eslint-disable complexity */
```

Comment:
This file needs a bit of refactor to avoid this. Will be done in a subsequent pr, just adding this to try to merge it before FF